components/portal/messages/partials/view__notifications.html

```diff
@@ -48,7 +48,7 @@
                 <!-- DISMISS BUTTON -->
                 <div class="notifications-list__action">
                   <md-button class="md-icon-button"
-                             ng-hide="notification.dismissible === false"
+                             ng-style="{'visibility' : notification.dismissible === false  ? 'hidden' : 'visible'}"
```

Comment:
I don't think that we need to make this change. I would keep ng-hide.

Reply:
This change will fix the alignment of action buttons. I could try other ways to fix it, but this seems easiest... What do you think?